server/api/router.go

```diff
@@ -22,7 +22,110 @@ import (
+	//HotWriteRegions is the api that may GET.
+	HotWriteRegions = "/api/v1/hotspot/regions/write"
+	//HotReadRegions is the api that may GET.
+	HotReadRegions = "/api/v1/hotspot/regions/read"
+	//HotStores is the api that may GET.
+	HotStores   = "/api/v1/hotspot/stores"
+	RegionByID  = "/api/v1/region/id"
+	RegionByKey = "/api/v1/region/key"
+	//Regions is the api that may GET.
+	Regions = "/api/v1/regions"
+	//ScanRegions is the api that may GET.
+	ScanRegions  = "/api/v1/regions/key"
+	StoreRegions = "/api/v1/regions/store"
+	//TopWriteFlow is the api that may GET.
+	TopWriteFlow = "/api/v1/regions/writeflow"
+	//TopReadFlow is the api that may GET.
+	TopReadFlow = "/api/v1/regions/readflow"
+	//TopConfVer is the api that may GET.
+	TopConfVer = "/api/v1/regions/confver"
+	//TopVersion is the api that may GET.
+	TopVersion = "/api/v1/regions/version"
+	//TopSize is the api that may GET.
+	TopSize = "/api/v1/regions/size"
+	//MissPeerRegions is the api that may GET.
+	MissPeerRegions = "/api/v1/regions/check/miss-peer"
+	//ExtraPeerRegions is the api that may GET.
+	ExtraPeerRegions = "/api/v1/regions/check/extra-peer"
+	//PendingPeerRegions is the api that may GET.
+	PendingPeerRegions = "/api/v1/regions/check/pending-peer"
+	//DownPeerRegions is the api that may GET.
+	DownPeerRegions = "/api/v1/regions/check/down-peer"
+	//OfflinePeer is the api that may GET.
+	OfflinePeer = "/api/v1/regions/check/offline-peer"
+	//EmptyRegion is the api that may GET.
+	EmptyRegion    = "/api/v1/regions/check/empty-region"
+	RegionSiblings = "/api/v1/regions/sibling"
+	//IncorrectNamespaceRegions is the api that may GET.
+	IncorrectNamespaceRegions = "/api/v1/regions/check/incorrect-ns"
+	//Version is the api that may GET.
+	Version = "/api/v1/version"
+	//Status is the api that may GET.
+	Status = "/api/v1/status"
+	//Members is the api that may GET.
+	Members       = "/api/v1/members"
+	MembersByName = "/api/v1/members/name"
+	MembersByID   = "/api/v1/members/id"
+	//Leader is the api that may GET.
+	Leader = "/api/v1/leader"
+	//LeaderResign is the api that may POST.
+	LeaderResign   = "/api/v1/leader/resign"
+	LeaderTransfer = "/api/v1/leader/transfer"
+	//StatsRegion is the api that may GET.
+	StatsRegion = "/api/v1/stats/region"
+	//Trends is the api that may GET.
+	Trends                = "/api/v1/trend"
+	HandleDropCacheRegion = "/api/v1/admin/cache/region"
+	//AdminLog is the api that may POST.
+	AdminLog = "/api/v1/admin/log"
+	//Healths is the api that may GET.
+	Healths = "/api/v1/health"
+	//Diagnose is the api that may GET.
+	Diagnose = "/api/v1/diagnose"
+	//DeprecatedHealths is the api that may GET.
+	DeprecatedHealths = "/health"
+	//DeprecatedDiagnose is the api that may GET.
+	DeprecatedDiagnose = "/diagnose"
+	Classifier         = "/api/v1/classifier/"
+	NameSpace          = "/table/namespaces"
+	NameSpaceTable     = "/table/namespaces/table"
+	NameSpaceMeta      = "/table/namespaces/meta"
+	StoreNameSpace     = "/table/store_ns"
+)
```

Comment:
I think it would be better to move comments to the end of line to make the layout more tidy.

Reply:
the rest LGTM.

Reply:
I have try to add comment `Ping = "/ping" // Ping is the api that may GET.`, but it can not pass the go lint test,

Reply:
😞 maybe we don't have to export all paths.